consumer.js

```diff
@@ -0,0 +1,14 @@
+"use strict"
```

Comment:
@Georgette Good point, and one I hadn't considered. Thinking more about this the consumer should probably be it's own microservice. It would give a clean scale point that way. Let's move consumer out to a `nodezoo-dequeue` repo as it's own thing. I'll create the repo.